params/config.go

```diff
@@ -230,6 +232,8 @@ type ChainConfig struct {
 	RandaoCompatibleBlock *big.Int        `json:"randaoCompatibleBlock,omitempty"` // RandaoCompatible activate block (nil = no fork)
 	RandaoRegistry        *RegistryConfig `json:"randaoRegistry,omitempty"`        // Registry initial states
 
+	PragueCompatibleBlock *big.Int `json:"pragueCompatibleBlock,omitempty"` // PragueCompatibl activate block (nil = no fork)
```

Comment:
Please place it below KaiaCompatibleBlock.

Our ChainConfigs struct is organized in the order of
- ChainID
- Mandatory hardforks
- Optional hardforks
- Other parameters

Reply:
Reordered. Thanks